src/styles/Signup.scss

```diff
@@ -0,0 +1,22 @@
+.sign-up-wrapper {
+  margin-top: 2rem;
+  text-align: center;
+
+  > * { display: block; }
```

Comment:
Add `&` before this and the next one. The same on the next element.

This is because you are saying that all elements inside `.sign-in-wrapper` have to be block elements, regarding if they are direct children or not. And I think you want just the direct ones.

The same for the next line. As it is right now, if you have an element with two or more elements inside, they will have a separation of `1rem` also.